admin/data/sql/migrations/20190815151308.undo.pupilgroup-vewpupilswithactivelivepins.sql

```diff
@@ -0,0 +1,24 @@
+ALTER VIEW [mtc_admin].[vewPupilsWithActiveLivePins] AS
+  SELECT
+         p.id,
+         p.foreName,
+         p.lastName,
+         p.middleNames,
+         p.dateOfBirth,
+         p.urlSlug,
+         p.school_id,
+         pin.val as pin,
+         cp.pinExpiresAt,
+         g.group_id
+  FROM [mtc_admin].[pupil] p
+         INNER JOIN [mtc_admin].[school] s ON p.school_id = s.id
+         LEFT JOIN  [mtc_admin].[pupilGroup] g ON g.pupil_id = p.id
```

Comment:
Why do we need to Join to [group] here?

Reply:
thats the undo query, which does in fact need the group table.